service/controller/clusterapi/v17/key/cluster.go

```diff
@@ -1,14 +1,65 @@
 package key
 
 import (
+	"github.com/giantswarm/cluster-operator/pkg/label"
```

Comment:
Sorry to nit but should be separate block.

Reply:
No worries. I always forget this as I'm so used to `goimports` to just figure out everything and when we slightly differ from this, it occasionally requires additional manual work (to be remembered). :+1: